docs/getting-started/certificate-pinning-best-practices.adoc

```diff
@@ -0,0 +1,30 @@
+= Overview
```

Comment:
is this file required as part of this PR?

Reply:
@wei-lee woops, no. I'll remove now.